br/pkg/lightning/backend/kv/session.go

```diff
@@ -315,7 +313,6 @@ func NewSession(options *encode.SessionOptions, logger log.Logger) *Session {
 		}
 	}
 	vars.StmtCtx.SetTimeZone(vars.Location())
-	vars.StmtCtx.SetTypeFlags(types.StrictFlags)
```

Comment:
This line is introduced in https://github.com/pingcap/tidb/pull/47842. I think it's a bug, right? @lcwangchao Please confirm.

Reply:
Seems to be a mistake, but introduced in previous PR: #47543 . In #47543 it overrides the type context settings before it...